### DISCUSSION TITLE
Automated 'Question/Fact of the day quizz' bounty

### DISCUSSION THREAD

**Discussion by hanseartic** [kalepail/stellar-quest-bounties#76] [Bounty Ideas]
We could have some simple UI/App where questers and bounty hunters could prove their knowledge about quests.
* A random not-yet answered question is presented to a logged-in user
* The user has limited time (really only sth. like a minute or 5) to answer
* The answer goes to validation by a lumenaut
* User get's rewarded for correct answer
* A User can answer max 1 question a day

Maybe a multiple-choice question would be easier to handle - otherwise harder to cope with different kinds of manipulation.

**Comment by kalepail** [comment]
This is a cool idea but seems pretty hard to implement without gamification. Would probably be better for a simpler SCF entry than a bounty imo.

**Reply by hanseartic** [reply]
That makes sense. Thank you for the evaluation.
